### PR TITLE
Removing unnecessary track decorators

### PR DESCRIPTION
**Pull request by satyasekharcvb**

### CODE REVIEW

File: force-app/main/default/lwc/harvestFieldMap/harvestFieldMap.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ export default class HarvestFieldMap extends LightningElement {
     @api harvestFields;
     @api recordId;
 
-    @track selectedFields = [];
-    @track selectedRecords = [];
+    selectedFields = [];
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/harvestFieldMap/harvestFieldMap.js]: Does this work?

File: force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js [hunk 2]
```diff
@@ -26,9 +26,9 @@ export default class HarvestFieldMapNav extends NavigationMixin(
     LightningElement
 ) {
     @api recordId;
-    @track error;
-    @track harvestField;
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js]: Is this property needed?

File: force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js [hunk 3]
```diff
@@ -26,9 +26,9 @@ export default class HarvestFieldMapNav extends NavigationMixin(
...
-    @track harvestFields;
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js]: Is this property needed?

File: force-app/main/default/lwc/harvestFieldMap/harvestFieldMap.js (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/harvestFieldMap/harvestFieldMap.js]: Yes, it works.

File: force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js (hunk 2 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js]: harvestField property is referenced

File: force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js (hunk 3 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js]: harvestFields property is not needed. There are are few other such properties which are not referenced. I will review the code, refactor and push those fixes in another PR.

**Pull request merged**